firmware/hw_layer/adc_inputs.cpp

```diff
@@ -498,32 +498,31 @@ static void configureInputs(void) {
 	 */
 
 	addChannel("MAP", engineConfiguration->map.sensor.hwChannel, ADC_FAST);
-	if (hasMafSensor()) {
```

Comment:
`hasMafSensor` just checks for unset channel - which is the same thing that `addChannel` does, so we don't need the check first.